CI failure: PYTEST on PR #72 (feat/issue-38-pr-issue-templates)

## CI Failure Detected
**PR:** #72
**Branch:** feat/issue-38-pr-issue-templates
**Failure type:** PYTEST
**Failing checks:** pytest (3.11),Coverage
**Run ID:** 24069716518

## Failure log (first 80 lines)
```
pytest (3.11)	Run tests	﻿2026-04-07T07:28:35.5941376Z ##[group]Run python -m pytest -v tests/
pytest (3.11)	Run tests	2026-04-07T07:28:35.5941722Z [36;1mpython -m pytest -v tests/[0m
pytest (3.11)	Run tests	2026-04-07T07:28:35.5968621Z shell: /usr/bin/bash -e {0}
pytest (3.11)	Run tests	2026-04-07T07:28:35.5968841Z env:
pytest (3.11)	Run tests	2026-04-07T07:28:35.5969079Z   pythonLocation: /opt/hostedtoolcache/Python/3.11.15/x64
pytest (3.11)	Run tests	2026-04-07T07:28:35.5969487Z   PKG_CONFIG_PATH: /opt/hostedtoolcache/Python/3.11.15/x64/lib/pkgconfig
pytest (3.11)	Run tests	2026-04-07T07:28:35.5969901Z   Python_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.15/x64
pytest (3.11)	Run tests	2026-04-07T07:28:35.5970271Z   Python2_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.15/x64
pytest (3.11)	Run tests	2026-04-07T07:28:35.5970638Z   Python3_ROOT_DIR: /opt/hostedtoolcache/Python/3.11.15/x64
pytest (3.11)	Run tests	2026-04-07T07:28:35.5971001Z   LD_LIBRARY_PATH: /opt/hostedtoolcache/Python/3.11.15/x64/lib
pytest (3.11)	Run tests	2026-04-07T07:28:35.5971601Z   DOCSTORE_URL: ***127.0.0.1:5432/ragpipe
pytest (3.11)	Run tests	2026-04-07T07:28:35.5971847Z ##[endgroup]
pytest (3.11)	Run tests	2026-04-07T07:28:36.8588832Z ============================= test session starts ==============================
pytest (3.11)	Run tests	2026-04-07T07:28:36.8589933Z platform linux -- Python 3.11.15, pytest-9.0.2, pluggy-1.6.0 -- /opt/hostedtoolcache/Python/3.11.15/x64/bin/python
pytest (3.11)	Run tests	2026-04-07T07:28:36.8590827Z cachedir: .pytest_cache
pytest (3.11)	Run tests	2026-04-07T07:28:36.8592051Z hypothesis profile 'ci' -> database=None, deadline=None, print_blob=True, derandomize=True, suppress_health_check=(HealthCheck.too_slow,)
pytest (3.11)	Run tests	2026-04-07T07:28:36.8593419Z rootdir: /home/runner/work/ragpipe/ragpipe
pytest (3.11)	Run tests	2026-04-07T07:28:36.8593981Z configfile: pyproject.toml
pytest (3.11)	Run tests	2026-04-07T07:28:36.8595137Z plugins: cov-7.1.0, hypothesis-6.151.11, anyio-4.13.0, asyncio-1.3.0
pytest (3.11)	Run tests	2026-04-07T07:28:36.8596826Z asyncio: mode=Mode.STRICT, debug=False, asyncio_default_fixture_loop_scope=None, asyncio_default_test_loop_scope=function
pytest (3.11)	Run tests	2026-04-07T07:28:39.1993670Z collecting ... collected 189 items / 1 error
pytest (3.11)	Run tests	2026-04-07T07:28:39.1994246Z 
pytest (3.11)	Run tests	2026-04-07T07:28:39.1994524Z ==================================== ERRORS ====================================
pytest (3.11)	Run tests	2026-04-07T07:28:39.1995253Z _____________________ ERROR collecting tests/test_live.py ______________________
pytest (3.11)	Run tests	2026-04-07T07:28:39.1996143Z ImportError while importing test module '/home/runner/work/ragpipe/ragpipe/tests/test_live.py'.
pytest (3.11)	Run tests	2026-04-07T07:28:39.2000235Z Hint: make sure your test modules/packages have valid Python names.
pytest (3.11)	Run tests	2026-04-07T07:28:39.2001045Z Traceback:
pytest (3.11)	Run tests	2026-04-07T07:28:39.2001710Z /opt/hostedtoolcache/Python/3.11.15/x64/lib/python3.11/importlib/__init__.py:126: in import_module
pytest (3.11)	Run tests	2026-04-07T07:28:39.2002441Z     return _bootstrap._gcd_import(name[level:], package, level)
pytest (3.11)	Run tests	2026-04-07T07:28:39.2002870Z            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
pytest (3.11)	Run tests	2026-04-07T07:28:39.2003207Z tests/test_live.py:20: in <module>
pytest (3.11)	Run tests	2026-04-07T07:28:39.2003489Z     import requests
pytest (3.11)	Run tests	2026-04-07T07:28:39.2003767Z E   ModuleNotFoundError: No module named 'requests'
pytest (3.11)	Run tests	2026-04-07T07:28:39.2004193Z =========================== short test summary info ============================
pytest (3.11)	Run tests	2026-04-07T07:28:39.2004578Z ERROR tests/test_live.py
pytest (3.11)	Run tests	2026-04-07T07:28:39.2004937Z !!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
pytest (3.11)	Run tests	2026-04-07T07:28:39.2005403Z =============================== 1 error in 2.71s ===============================
pytest (3.11)	Run tests	2026-04-07T07:28:39.4074084Z ##[error]Process completed with exit code 2.
Coverage	Run tests with coverage	﻿2026-04-07T07:28:44.8237921Z ##[group]Run python -m pytest --cov=ragpipe --cov-report=xml --cov-report=term-missing tests/
Coverage	Run tests with coverage	2026-04-07T07:28:44.8238610Z [36;1mpython -m pytest --cov=ragpipe --cov-report=xml --cov-report=term-missing tests/[0m
```

## Detected by
Nemotron CI watcher — Tue Apr  7 03:30:47 AM EDT 2026

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -25,7 +25,7 @@ dependencies = [
 ]
 
 [project.optional-dependencies]
-dev = ["pytest", "pytest-asyncio", "pytest-cov", "ruff", "hypothesis"]
+dev = ["pytest", "pytest-asyncio", "pytest-cov", "ruff", "hypothesis", "requests"]
 
 [project.scripts]
 ragpipe = "ragpipe.app:main"

